docs/json_story_format/json_story_format.md

```diff
@@ -20,20 +22,26 @@ Example:
             house.json
     stats.json
 ```
+
 If there is more than one layer of folders, folder names will be added before a name which is used to access it, for example: `mine_big_room`
```

Comment:
is this just a convention or a limitation of the engine? i think this shouldnt be forced.

Reply:
I guess we shouldn't have this, because if we change the folder of the file, it might breaks the story.

Reply:
yeah, leave it up to the person implementing the story so its more of a convention thing.